spec.md

```diff
@@ -124,27 +124,44 @@ Typically, the first step in pulling an artifact is to retrieve the manifest. Ho
 
 ##### Pulling manifests
 
-To pull a manifest, perform a `GET` request to a url in the following form:
+To pull a manifest, perform a `GET` request to a URL in the following form:
 `/v2/<name>/manifests/<reference>` <sup>[end-3](#endpoints)</sup>
 
 `<name>` refers to the namespace of the repository. `<reference>` MUST be either (a) the digest of the manifest or (b) a tag name.
 The `<reference>` MUST NOT be in any other format.
 
 A GET request to an existing manifest URL MUST provide the expected manifest, with a response code that MUST be `200 OK`.
+A successful response SHOULD contain the digest of the uploaded blob in the header `OCI-Content-Digest`. For legacy
+reasons, the digest MAY be contained in the header `Docker-Content-Digest` instead. Either header is OPTIONAL.
```

Comment:
Would it make sense to have a backwards compatibility section somewhere in the spec? It could include `Docker-Content-Digest` and https://docs.docker.com/registry/spec/manifest-v2-2/#backward-compatibility

Some precedent in the image-spec: https://github.com/opencontainers/image-spec/blob/master/media-types.md#compatibility-matrix

Reply:
Yes, I think that's a good suggestion. It's good to keep "legacy" and "specification" separate, but given that we're still somewhat in a transition, it's hard to _fully_ ignore the old standards (it's not just registries, but many older clients are still active in the wild). Having sections dedicated to guidelines for "how to get a smooth transition" / "things to consider for backward compatibility" seems like a nice middle ground.

Reply:
Based on several conversations about this: a new OCI- header will not be introduced. A subsequent release may introduce this header if it becomes necessary